charts/yawol-controller/templates/yawol-controller.yaml

```diff
@@ -64,9 +70,6 @@ spec:
         args:
```

Comment:
I think we should add this to all 3 controllers not only the lb and lbm controller